samples/nrf9160/http_application_fota/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _http_application_fota_sample:
+
+nRF9160: HTTP application update
+################################
+
+Th HTTP application update sample demonstrates how to do a basic FOTA (firmware over-the-air) update.
+It uses the :ref:`lib_download_client` library to download a file from a remote server and writes it to flash.
+
+
+Overview
+********
+
+The sample connects to an HTTP server to download a firmware image.
+You must provide the image file and configure where it can be downloaded.
+See `Specifying the image file`_ for more information.
+
+By default, the image is saved to slot 1 (see :ref:`ug_bootloader_architecture`).
+The downloaded file is not checked or authenticated, and no further processing is done.
```

Comment:
So this means it is ONLY downloaded, and it's not really a sample for an update?

Reply:
It is downloaded and programmed to flash. The plan was to use MCUBoot but I removed that from the README file in the last minute since we were unable to use it with MCUBoot. We are debugging MCUBoot as I write this and if we get it to work the file to be downloaded needs to be signed for MCUBoot to accept it.

Reply:
So maybe like this?

The downloaded file is not checked or authenticated.
It is only copied to slot 1, replacing the current firmware there.